lib/net/ssh/transport/session.rb

```diff
@@ -61,12 +61,12 @@ def initialize(host, options={})
       @bind_address = options[:bind_address] || nil
       @options = options
 
-      debug { "establishing connection to #{@host}:#{@port}" }
-
       @socket =
         if (factory = options[:proxy])
+          debug { "establishing connection to proxy" }
```

Comment:
wouldn't make sense to print host port too? like `proxy #{@host}:#{@port}`? I guess any more clarification could be done by `open` method in proxy.